demo/web1.html

```diff
@@ -9,10 +9,10 @@
 
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css">
   <link rel="stylesheet" href="demo.css" />
-
+  
   <script type="module" src="https://unpkg.com/ionicons@4.5.10-0/dist/ionicons/ionicons.esm.js"></script>
   <script nomodule="" src="https://unpkg.com/ionicons@4.5.10-0/dist/ionicons/ionicons.js"></script>
-
+  <script src="../src/gridstack-poly.js"></script>
```

Comment:
should be ../dist as poly should be copies to the build folder as well (or needs to)

Reply:
ok